spec/models/applicant_spec.rb

```diff
@@ -102,16 +102,47 @@
   end
 
   describe "#age" do
-    subject { legal_aid_application.applicant.age }
+    subject(:age) { legal_aid_application.applicant.age }
 
-    let(:legal_aid_application) { build(:legal_aid_application, :with_transaction_period, :with_applicant) }
+    let(:applicant) { build(:applicant, date_of_birth: 48.years.ago) }
+
+    let(:legal_aid_application) do
+      build(:legal_aid_application, :with_transaction_period, applicant:)
+    end
 
     it "returns the age of the applicant" do
-      expect(subject).to be_a(Integer)
+      expect(age).to be_a(Integer)
```

Comment:
I'm not convinced this tests anything useful. 

Consider this example

```ruby
# spec/models/applicant_spec.rb
describe "#age" do
  let(:applicant) { build(:applicant, date_of_birth: 48.years.ago) }

  it "returns the age of the applicant" do
    expect(age).to be_a(Integer)
  end 
end

# app/models/applicant.rb
def age
  17
end
```

The test would still pass, but the age is clearly wrong.

Expecting the method to return an integer might have been sufficient when the method contained no logic besides calling `AgeCalculator` (which is unit tested itself), but I think we should be more specific now the method actually does something.